### PR TITLE
chore: remove unused exports

### PR DESCRIPTION
**Pull request by clegirar**

### CODE REVIEW

**Comment by WaDadidou** [comment]
Could you rename `RiotGameScreen.web.tsx` as `RiotGameScreen.tsx` since this file is removed ? https://github.com/TERITORI/teritori-dapp/blob/ce0d7ea427b993e0dc4f1c8a8006bb821933d445/packages/screens/RiotGame/RiotGameScreen.tsx
(And change the import in `Navigator.tsx`)

**Comment by WaDadidou** [comment]
In https://github.com/TERITORI/teritori-dapp/blob/68f8f691930c3a669e9cc037392694edacfdce30/packages/components/socialFeed/SocialActions/ShareButton.tsx#L10, could you rename `FeedPostShareModalProps` as `ShareButtonProps` (Or remove this interface, cause there is only one property), since this file is removed: https://github.com/TERITORI/teritori-dapp/blob/ce0d7ea427b993e0dc4f1c8a8006bb821933d445/packages/components/socialFeed/SocialActions/FeedPostShareModal.tsx

**Comment by WaDadidou** [comment]
Please fix lint-and-build and update the branch with origin/main

**Review by WaDadidou — CHANGES_REQUESTED** [review]

File: packages/screens/DAppStore/apps/toripunks/query/useHistoryData.ts
[the remark this answers is not in the document]
Comment (clegirar) [reply packages/screens/DAppStore/apps/toripunks/query/useHistoryData.ts:1]: Oh yeah you right 👍

File: packages/screens/DAppStore/apps/toripunks/query/userHistoryMock.ts
[the remark this answers is not in the document]
Comment (clegirar) [reply packages/screens/DAppStore/apps/toripunks/query/userHistoryMock.ts:1]: Yes you right

File: packages/components/gradientText/index.ts
[the remark this answers is not in the document]
Comment (clegirar) [reply packages/components/gradientText/index.ts:1]: Which tool give you this error ? Because i have nothing that say that ?

**Comment by clegirar** [comment]
If it's possible i want to rebase on main after merging Audio Player to have just one nightmare to resolve conflicts 🙃

**Review by omniwired — DISMISSED** [review]

File: packages/components/gradientText/index.ts
[the remark this answers is not in the document]
Comment (omniwired) [reply packages/components/gradientText/index.ts:1]: it was happening when trying to build the project

**Pull request merged**